packages/admin/admin/src/index.ts

```diff
@@ -79,12 +79,14 @@ export { FinalFormCheckbox, FinalFormCheckboxProps } from "./form/Checkbox";
 export { Field, FieldProps } from "./form/Field";
 export { FieldContainer, FieldContainerClassKey, FieldContainerComponent, FieldContainerProps } from "./form/FieldContainer";
 export { CheckboxField, CheckboxFieldProps } from "./form/fields/CheckboxField";
+export { CurrencyField } from "./form/fields/CurrencyField";
 export { SearchField, SearchFieldProps } from "./form/fields/SearchField";
 export { SelectField, SelectFieldProps } from "./form/fields/SelectField";
 export { SwitchField, SwitchFieldProps } from "./form/fields/SwitchField";
 export { TextAreaField, TextAreaFieldProps } from "./form/fields/TextAreaField";
 export { TextField, TextFieldProps } from "./form/fields/TextField";
 export { FinalFormContext, FinalFormContextProvider, FinalFormContextProviderProps, useFinalFormContext } from "./form/FinalFormContextProvider";
+export { FinalFormCurrencyInput } from "./form/FinalFormCurrencyInput";
```

Comment:
Please also export the props.

Reply:
Please also export the final-form component and it's props, as is done with other field components.